src/core/worker.js

```diff
@@ -487,6 +491,16 @@ class WorkerMessageHandler {
       });
     });
 
+    handler.on("GetPageXfa", function wphSetupGetXfa({ pageIndex }) {
+      return pdfManager.getPage(pageIndex).then(function (page) {
+        return page.xfaData;
```

Comment:
I think we need an ensure `pdfManager.ensure(page, xfaData` since this could potentially throw if the data isn't available.